db/migrate/20160709205224_add_check_to_games.rb

```diff
@@ -0,0 +1,5 @@
+class AddCheckToGames < ActiveRecord::Migration
+  def change
+    add_column :games, :check, :string 
```

Comment:
Do you think we need it? Whats the reason to save it in the database? Its a very dynamic state, isn't it?